robottelo/datafactory.py

```diff
@@ -277,17 +317,14 @@ def valid_org_names_list():
 
 @datacheck
 def valid_usernames_list():
-    """Returns a list of valid user names.
-
-    Note: utf8 data is being rejected by the satellite server so it is not
-    included
-
-    """
+    """Returns a list of valid user names."""
     return [
         gen_string("latin1"),
         gen_string("alpha"),
         gen_string("alphanumeric"),
         gen_string("numeric"),
+        gen_string("utf8"),
+        gen_string('cjk'),
```

Comment:
One more suggestion from me - better not to stick to a constant length of 10 chars, but use random lengths, e.g. from 1 to 100 (i believe boundary for logins is 100 chars).